virttest/utils_test/__init__.py

```diff
@@ -240,10 +240,10 @@ def get_time(session, time_command, time_filter_re, time_format):
                 msg = "Fail to get guest time offset. Command "
                 msg += "'%s', output: %s" % (time_command, output)
                 raise exceptions.TestError(msg)
-            if re.match('PM', guest_time):
+            if re.search('PM', guest_time):
                 hour = re.findall('\d+ (\d+):', guest_time)[0]
                 hour = str(int(hour) + 12)
-                guest_time = re.sub('\d+\s\d+:', "\d+\s%s:" % hour,
+                guest_time = re.sub('\d+/\s\d+:', "\d+\s%s:" % hour,
```

Comment:
@luckyh 
Yes, it want to add 12 hours when get time with "PM",  the previous pattern can not get the correct result:
>>> guest_time="3/30/2017 3:28:47 PM"                                               
>>> if re.search('PM',guest_time):                                                  
...     hour = re.findall('\d+ (\d+):', guest_time)[0]                              
...     hour = str(int(hour) + 12)                                                  
...     guest_time = re.sub('\d+\s\d+:', '\d+\s %s:' % hour, guest_time)[:-3]
...     print guest_time
... 
3/30/\d+\s 15:28:47

And, of course, it is my careless when fix the above issue but ignore the actual time.
patch updated , please help review again, thanks :)